src/platform/platform_internal.h

```diff
@@ -633,7 +638,13 @@ typedef struct CXPLAT_SOCKET {
     uint8_t Uninitialized : 1;
     uint8_t Freed : 1;
 
-    uint8_t UseTcp : 1;                  // Quic over TCP
+    //
+    // This flag determines whether or not we instantiate an auxiliary TCP
+    // socket in XDP mode. For clients, we either only create an auxiliary
+    // TCP socket, or per-proc UDP sockets. For servers, we always create
+    // per-proc UDP sockets, and optionally create an auxiliary TCP socket.
+    //
+    uint8_t UseTcp : 1;
```

Comment:
nit: Would it make sense to rename this "TcpSocketAvailable" / "TcpSocketInstantiated" / ...? 
Since using QTIP depends on the route for server, is could be confusing to see "UseTcp" and then not use TCP :)

But that's nitpicky, feel free to ignore.

Reply:
Yeah "useTcp" is a misleading and confusing name. I updated it.